Incorporate improvements to original provisioning (#3922)
theforeman/foreman-documentation: guides/common/modules/snip_warning-destroy-vm-on-host-delete.adoc

[WARNING]
====

> By default, if a host entry that is associated with a virtual machine is deleted, the virtual machine will remain on the compute resource.

Reviewer: ```suggestion
By default, deleting a host entry associated with a virtual machine does not delete the virtual machine on the compute resource.
```
Suggestion for removing passive voice and future tense.

Author: A small suggestion:
```suggestion
By default, deleting a host entry associated with a virtual machine does not destroy the virtual machine on the compute resource.
```

If you want {Project} to destroy virtual machines on the compute resource when the associated host entry is deleted, navigate to *Administer* > *Settings* > *Provisioning* and set the *Destroy associated VM on host delete* setting to *Yes*.

> To avoid destroying virtual machines in this situation, disassociate the virtual machine from {Project} without removing it from the compute resource or keep the setting set to *No*.

Reviewer: I don't think you need this sentence because you have already explained the default behavior.
